### PR TITLE
My solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**
add_comments_no_login
commet_login
post list
post_detail

### CODE REVIEW

File: blog/models.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(to=User,
```
Comment (YuriiKindrat) [inline blog/models.py:10]: it is better to use AUTH_USER_MODEL here

File: blog/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    title = models.CharField(max_length=255)
+    content = models.CharField(max_length=255)
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ["title"]
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(to=User,
```
Comment (YuriiKindrat) [inline blog/models.py:22]: same

File: templates/base.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+      {% block pagination %}
+        {% include "includes/pagination.html" %}
+      {% endblock %}
+
+    </div>
+  </div>
+</div>
+</body>
+</html>
```
Comment (YuriiKindrat) [inline templates/base.html:41]: no blank line at the end of file

Comment (YuriiKindrat) [reply templates/base.html:41]: fix this everywhere

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**